packages/contracts-ops/contracts/test/RebootHome.t.sol

```diff
@@ -15,6 +15,8 @@ contract HomeRebootTest is RebootTest, HomeTest {
         home = HomeHarness(address(getHome(localDomainName)));
         upgradeHomeHarness();
         updaterManager = getUpdaterManager(localDomainName);
+        // check fork setUp
+        assertEq(home.updater(), updaterAddr);
```

Comment:
no assertions in `setUp`. due to a quirk in Foundry, these will not cause tests to fail.. instead add a `test_setUp` that makes this assertion

Reply:
oh weird, i thought that was a bug that they had fixed?

Reply:
I think it's fixed, but as a general pattern, we aligned that it's better to create a test function that doesn't do any state changes but tests the states of the setUp.